package.json

```diff
@@ -37,6 +34,7 @@
         "@nestjs/serve-static": "^2.1.3",
         "@nestjs/swagger": "^6.3.0",
         "@node-rs/argon2": "^1.4.0",
+        "@node-rs/argon2-linux-x64-gnu": "^1.7.2",
```

Comment:
This can't be added. It's only meant for it to run on linux. Add it to the DevDependencies for now